Unix/base/batch.c

```diff
@@ -35,7 +40,11 @@ void* Batch_Get(
     /* Allocate a new page and carve new block from that page */
     if (size8 <= BLOCK_SIZE && self->numPages < self->maxPages)
     {
+#if defined(CONFIG_ENABLE_DEBUG) && defined(CONFIG_OS_LINUX)
+        Page* page = __PAL_Malloc(file, line, func, BATCH_PAGE_SIZE);
```

Comment:
Feel like file, line, and func should be passed after BATCH_PAGE_SIZE. BATCH_PAGE_SIZE is mandatory, the others get passed based on mode. Ditto for all below.

Reply:
I'm following same convention as some existing functions.  See declaration of __PAL_Malloc and __PAL_Calloc in pal/palcommon.h.   If you feel strongly about this, I can move code around.